tycho-its/projects/target.artifact.junit/target-platform/target-platform.target

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?>
+<target includeMode="feature" name="target-platform">
+  <locations>
+    <location includeDependencyDepth="none" includeSource="true" missingManifest="generate" type="Maven">
+      <dependencies>
```

Comment:
with recent tycho, adding thigs manually to the target togehter with a Junit5 container is superflous.

Reply:
@laeubi so starting with Tycho 4, I can omit JUnit 5 from the target platform, that's the meaning?
Basically JUnit 5 is going to be resolved automatically if it's in the `.classpath` file?

Reply:
Yes it was also previously resolved (but only from the target) now it is automatically resolved from maven if not already in the target.

Reply:
@laeubi I'll try removing it from the target then! Thanks.
Although by how you're describing it, is seems keeping it as it is - in the target - should work.

Reply:
There are two demo projects here:
https://github.com/eclipse-tycho/tycho/tree/master/demo/testing/surefire

so in general junit 5 _should_ work, so I just assumed that probably something with the target setup is wrong.

Reply:
Thanks - those demos are only using maven-surefire-plugin I think so I've been looking at https://github.com/eclipse-tycho/tycho/tree/master/demo/testing/tycho/standalone

Reply:
It all depends on your use-case, setup and what you want to archive see:
https://tycho.eclipseprojects.io/doc/master/TestingBundles.html